.github/workflows/continuous-integration.yml

```diff
@@ -152,7 +212,364 @@ jobs:
+
+        options: >-
+          --health-cmd "mysqladmin ping --silent"
+          -e MYSQL_ALLOW_EMPTY_PASSWORD=yes
+          -e MYSQL_DATABASE=doctrine_tests
+          ${{ matrix.custom-entrypoint }}
+
+        ports:
+          - "3306:3306"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2"
+        with:
+          fetch-depth: 2
+
+      - name: "Install PHP"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          php-version: "${{ matrix.php-version }}"
+          coverage: "pcov"
+          extensions: "${{ matrix.driver }}"
+
+      - name: "Cache dependencies installed with composer"
+        uses: "actions/cache@v2"
+        with:
+          path: "~/.composer/cache"
+          key: "php-${{ matrix.php-version }}-composer-locked-${{ hashFiles('composer.lock') }}"
+          restore-keys: "php-${{ matrix.php-version }}-composer-locked-"
+
+      - name: "Install dependencies with composer"
+        run: "composer install --no-interaction --no-progress --no-suggest"
+
+      - name: "Copy TLS-related files"
+        run: 'docker cp "${{ job.services.mysql.id }}:/var/lib/mysql/ca.pem" . && docker cp "${{ job.services.mysql.id }}:/var/lib/mysql/client-cert.pem" . && docker cp "${{ job.services.mysql.id }}:/var/lib/mysql/client-key.pem" .'
+        if: "${{ endsWith(matrix.config-file-suffix, 'tls') }}"
+
+      - name: "Run PHPUnit"
+        run: "vendor/bin/phpunit -c ci/github/phpunit/${{ matrix.driver }}${{ matrix.config-file-suffix }}.xml --coverage-clover=coverage.xml"
+
+      - name: "Upload Code Coverage"
+        uses: "codecov/codecov-action@v1"
+
+  phpunit-mssql:
+    name: "PHPUnit with SQL Server"
+    runs-on: "ubuntu-20.04"
+    needs: "phpunit-smoke-check"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.3"
+          - "7.4"
+        driver:
+          - "sqlsrv"
+          - "pdo-sqlsrv"
+        collation:
+          - "Latin1_General_100_CI_AS"
+        include:
+          - collation: "Latin1_General_100_CS_AS"
+            php-version: "7.4"
+            driver: "sqlsrv"
+          - collation: "Latin1_General_100_CS_AS"
+            php-version: "7.4"
+            driver: "pdo-sqlsrv"
+
+    services:
+      mssql:
+        image: "microsoft/mssql-server-linux:2017-latest"
+        env:
+          ACCEPT_EULA: "Y"
+          SA_PASSWORD: "Doctrine2018"
+          MSSQL_COLLATION: "${{ matrix.collation }}"
+
+        options: >-
+          --health-cmd "echo quit | /opt/mssql-tools/bin/sqlcmd -S 127.0.0.1 -l 1 -U sa -P Doctrine2018"
+
+        ports:
+          - "1433:1433"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2"
+        with:
+          fetch-depth: 2
+
+      - name: "Install PHP"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          php-version: "${{ matrix.php-version }}"
+          coverage: "pcov"
+          tools: "pecl"
+          extensions: "${{ matrix.driver }}-5.7.0preview"
```

Comment:
I have to find a way to use an underscore here.

Reply:
Either that or I revert to `pdo_`

Reply:
I cannot find a clean solution @morozov … can you? Please note that inside the phpunit files, the `db_driver` values uses underscore. I'll revert to underscores (same for `ibm-db2`)

Reply:
Let's use underscore for PHP extensions and the corresponding filename fragments.

Reply:
Done